Make failing for new gnuRadio install on ubuntu for interpolating_resampler.cc

On checking out the latest version (commit: bd611224ba4c2356c1ad4c3359781682c5c121da), I get the following errors on make:

make[2]: *** [gr-digital/lib/CMakeFiles/gnuradio-digital.dir/interpolating_resampler.cc.o] Error 1
make[1]: *** [gr-digital/lib/CMakeFiles/gnuradio-digital.dir/all] Error 2

Error:
--------
gnuradio/gr-digital/lib/interpolating_resampler.cc:392:70:   instantiated from here
/usr/include/c++/4.6/bits/stl_vector.h:1080:4: error: invalid conversion from ‘int’ to ‘std::vector<gr::filter::kernel::fir_filter_fff*>::value_type {aka gr::filter::kernel::fir_filter_fff*}’ [-fpermissive]


usr/include/c++/4.6/bits/stl_vector.h:1122:7: error:   initializing argument 2 of ‘void std::vector<_Tp, _Alloc>::_M_fill_initialize(std::vector<_Tp, _Alloc>::size_type, const value_type&) [with _Tp = gr::filter::kernel::fir_filter_fff*, _Alloc = std::allocator<gr::filter::kernel::fir_filter_fff*>, std::vector<_Tp, _Alloc>::size_type = unsigned int, std::vector<_Tp, _Alloc>::value_type = gr::filter::kernel::fir_filter_fff*]’ [-fpermissive]

--- a/gr-digital/lib/interpolating_resampler.cc
+++ b/gr-digital/lib/interpolating_resampler.cc
@@ -291,11 +291,8 @@ namespace gr {
         // N.B. We create an extra final row for an offset of 1.0, because it's
         // easier than dealing with wrap around from 0.99... to 0.0 shifted
         // by 1 tap.
-        d_filters = std::vector<filter::kernel::fir_filter_ccf*>(d_nfilters + 1,
-                                                                 NULL);
-        d_diff_filters = std::vector<filter::kernel::fir_filter_ccf*>(
-                                                                 d_nfilters + 1,
-                                                                 NULL);
+        d_filters = std::vector<filter::kernel::fir_filter_ccf*>(d_nfilters + 1);
+        d_diff_filters = std::vector<filter::kernel::fir_filter_ccf*>(d_nfilters + 1);
 
         std::vector<float> t(NTAPS, 0);
         int incr = NSTEPS/d_nfilters;
@@ -388,11 +385,8 @@ namespace gr {
         // N.B. We create an extra final row for an offset of 1.0, because it's
         // easier than dealing with wrap around from 0.99... to 0.0 shifted
         // by 1 tap.
-        d_filters = std::vector<filter::kernel::fir_filter_fff*>(d_nfilters + 1,
-                                                                 NULL);
-        d_diff_filters = std::vector<filter::kernel::fir_filter_fff*>(
-                                                                 d_nfilters + 1,
-                                                                 NULL);
+        d_filters = std::vector<filter::kernel::fir_filter_fff*>(d_nfilters + 1);
+        d_diff_filters = std::vector<filter::kernel::fir_filter_fff*>(d_nfilters + 1);
 
         std::vector<float> t(NTAPS, 0);
         int incr = NSTEPS/d_nfilters;
@@ -541,11 +535,8 @@ namespace gr {
         // N.B. We create an extra final row for an offset of 1.0, because it's
         // easier than dealing with wrap around from 0.99... to 0.0 shifted
         // by 1 tap.
-        d_filters = std::vector<filter::kernel::fir_filter_ccf*>(d_nfilters + 1,
-                                                                 NULL);
-        d_diff_filters = std::vector<filter::kernel::fir_filter_ccf*>(
-                                                                 d_nfilters + 1,
-                                                                 NULL);
+        d_filters = std::vector<filter::kernel::fir_filter_ccf*>(d_nfilters + 1);
+        d_diff_filters = std::vector<filter::kernel::fir_filter_ccf*>(d_nfilters + 1);
 
         m = taps.size();
         n = diff_taps.size();
@@ -709,11 +700,8 @@ namespace gr {
         // N.B. We create an extra final row for an offset of 1.0, because it's
         // easier than dealing with wrap around from 0.99... to 0.0 shifted
         // by 1 tap.
-        d_filters = std::vector<filter::kernel::fir_filter_fff*>(d_nfilters + 1,
-                                                                 NULL);
-        d_diff_filters = std::vector<filter::kernel::fir_filter_fff*>(
-                                                                 d_nfilters + 1,
-                                                                 NULL);
+        d_filters = std::vector<filter::kernel::fir_filter_fff*>(d_nfilters + 1);
+        d_diff_filters = std::vector<filter::kernel::fir_filter_fff*>(d_nfilters + 1);
 
         m = taps.size();
         n = diff_taps.size();

